modules/log6x-rn-6.0.1.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-6-0-1_{context}"]
+= Logging 6.0.1
+This release includes link:https://access.redhat.com/errata/RHSA-2024:8315[OpenShift Logging Bug Fix Release 6.0.1].
+
+// 4.17
+
+[id="openshift-logging-6-0-1-bug-fixes_{context}"]
+== Bug fixes
+
+* With this update, the default memory limit for the collector has been increased from 1024Mi to 2024Mi. However, users should always adjust their resource limits according to their cluster specifications and needs. (link:https://issues.redhat.com/browse/LOG-6180[LOG-6180])
```

Comment:
I think the general rule of thumb for units of measurement is a space after the number: https://www.ibm.com/docs/en/ibm-style?topic=measurement-numbers#ranges

```suggestion
* With this update, the default memory limit for the collector has been increased from 1024 Mi to 2024 Mi. However, users should always adjust their resource limits according to their cluster specifications and needs. (link:https://issues.redhat.com/browse/LOG-6180[LOG-6180])
```

But maybe there's another rule that conflicts that!

Reply:
Fixed, ty.